imports/plugins/included/default-theme/client/styles/select.less

```diff
@@ -25,3 +25,7 @@
   width: 44px;
   pointer-events: none;
 }
+
+.Select-menu-outer {
+  z-index: 9999;
```

Comment:
Try a zindex value from the variables. Want to avoid a war of z-indicies.

in `variables.less`

```less
@zindex-navbar:            1000;
@zindex-dropdown:          1000;
@zindex-popover:           1060;
@zindex-tooltip:           1070;
@zindex-navbar-fixed:      1030;
@zindex-modal-background:  1040;
@zindex-modal:             1050;
```